packages/react-dom/src/shared/ReactDOMUnknownPropertyHook.js

```diff
@@ -68,7 +68,7 @@ if (__DEV__) {
       return true;
     }
 
-    if (lowerCasedName.indexOf('on') === 0 && lowerCasedName.length > 2) {
+    if (lowerCasedName.length > 2 && lowerCasedName.indexOf('on') === 0) {
```

Comment:
This one is not necessary. We're guarding against `[]` access outside of bounds in particular. Here it doesn't happen.

Reply:
I see. Sorry I accidentally messes up with the git, should I squash them?

Reply:
Doesn't matter, we squash on merge anyway.